File: app/src/main/java/com/example/myapplication/data/repository/AidlRoomUserRepository.java
package com.example.myapplication.data.repository;

import android.content.Context;

import com.example.myapplication.data.remote.service.UserAidlRoomService;

public class AidlRoomUserRepository extends AidlUserRepository {

    public AidlRoomUserRepository(Context context) {
        super(context, UserAidlRoomService.class);
    }
}

File: app/src/main/java/com/example/myapplication/data/repository/AidlRoomUserRepository.kt
package com.example.myapplication.data.repository

import android.content.Context
import com.example.myapplication.data.remote.service.UserAidlRoomService

class AidlRoomUserRepository(
    context: Context,
) : AidlUserRepository(
    context = context,
    serviceClass = UserAidlRoomService::class.java,
)
